### PR TITLE
Separate util functions

### PR DESCRIPTION
**Pull request by szledan**
Fixes: #121.

New structures:
  * bounding-box,
  * color,
  * float,
  * float-point,
  * line-types,
  * region,
  * state,
  * transform,
  * vec4.

Signed-off-by: Szilard Ledan <szledan@gmail.com>

### CODE REVIEW

File: src/utils/gepard-types.cpp [hunk 1]
```diff
@@ -24,310 +24,3 @@
  */
 
 #include "gepard-types.h"
```
Comment (kkristof) [inline src/utils/gepard-types.cpp]: This file seems empty after this change, I think this sould be removed.

Comment (szledan) [reply src/utils/gepard-types.cpp]: You are right. We will remove these files (`gepard-types.*`) as soon as possible. (Issue: #126 )

Comment (kkristof) [reply src/utils/gepard-types.cpp]: ok

**Comment by kkristof** [comment]
+1 LGTM, thanks

**Pull request merged**